src/digidoc-tool.cpp

```diff
@@ -830,6 +836,73 @@ static int create(int argc, char* argv[])
 
 
 /**
+ * Create new container.
+ *
+ * @param argc number of command line arguments.
+ * @param argv command line arguments.
+ * @return EXIT_FAILURE (1) - failure, EXIT_SUCCESS (0) - success
+ */
+static int createPatch(int argc, char* argv[])
+{
```

Comment:
This method does a lot of the same things as signContainer(), as in everything except the for loop, the .asice check and the dontSign flag check, that means a whole lot of code duplication here :(
Could just use the call to signContainer() inside the loop or extract the common parts to separate methods